vscode/src/webviewMessageHandler.ts

```diff
@@ -1,16 +1,43 @@
 import * as vscode from "vscode";
+import { ExtensionState } from "./extensionState";
+import { KonveyorGUIWebviewViewProvider } from "./KonveyorGUIWebviewViewProvider";
+
+export function setupWebviewMessageListener(
+  webview: vscode.Webview,
+  state: ExtensionState,
+  provider: KonveyorGUIWebviewViewProvider,
+) {
+  const incidentDataString = state.extensionContext.workspaceState.get<string>(
+    "incidentData",
+    "[]",
+  );
+  const incidentData = JSON.parse(incidentDataString);
+
+  webview.onDidReceiveMessage(async (message) => {
+    const fileUri = vscode.Uri.parse(message.file);
```

Comment:
This is really cool to see. Should this particular line be in the `openFile` case to prevent failure when the `message.file` would not be defined?